src/toolchain.rs

```diff
@@ -178,6 +178,25 @@ impl Toolchain {
         }),
     };
 
+    /// Returns whether or not this toolchain is needed by rustwide itself.
+    ///
+    /// This toolchain is used for doing things like installing tools.
+    ///
+    /// ```rust
+    /// # use rustwide::Toolchain;
+    /// let tc = Toolchain::dist("stable-x86_64-unknown-linux-gnu");
+    /// assert!(tc.is_needed_by_rustwide());
+    /// let tc = Toolchain::dist("nightly-x86_64-unknown-linux-gnu");
+    /// assert!(!tc.is_needed_by_rustwide());
+    /// ```
+    pub fn is_needed_by_rustwide(&self) -> bool {
+        match &self.inner {
+            ToolchainInner::Dist(dist) => dist.name.starts_with(MAIN_TOOLCHAIN_NAME),
+            #[cfg(feature = "unstable-toolchain-ci")]
+            _ => false,
+        }
```

Comment:
Hmm, wouldn't `self == Self::MAIN` work too?

Reply:
I don't think so.  In the crater use case, it is iterating over the actual toolchain names, so the `Toolchian::Dist` name would be comparing `stable-x86_64-unknown-linux-gnu` == `stable`.  I think it would need something like rustup's `PartialToolchainDesc` for comparing partial names to full names.

Reply:
Ugh forgot about that. Then just add a changelog line for this new method and I'll merge and release.

Reply:
Done.